builtins/web/abort/weak-index-set.h

```diff
@@ -31,7 +30,7 @@ class WeakIndexSet {
   }
 
   WeakVec &items() { return items_;}
-  const WeakVec &items() const { return items_; }
+  [[nodiscard]] [[nodiscard]] [[nodiscard]] const WeakVec &items() const { return items_; }
```

Comment:
I hope we don't need to be quite as insistent here ;)
```suggestion
  [[nodiscard]] const WeakVec &items() const { return items_; }
```